pages/index.js

```diff
@@ -1,13 +1,32 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import styled from "styled-components";
+import axios from "axios";
 
-import LoginForm from "../src/components/login-form";
+import MemorizedLoginForm from "../src/components/login-form";
+import MemorizedVoteForm from "../src/components/vote-form";
 
 export default function Home() {
+  const [isLoggedIn, setLoginStatus] = useState(false);
```

Comment:
```suggestion
  const [isLoggedIn, setIsLoggedIn] = useState(false);
```
```useState()```로 state를 선언할 때, 보통 그 state의 set 함수를 set 뒤에 state 이름을 붙여 네이밍 해줍니다. 그 state에 값을 set하는 함수임을 명시적으로 나타내기 위해서요!